helloworld-mutual-ssl-secured/configure-client-truststore.cli

```diff
@@ -0,0 +1,11 @@
+#extract the certificate for the server keystore into a file called serverCert.crt
+/subsystem=elytron/key-store=applicationKS:export-certificate(alias=server, path=serverCert.crt, relative-to=jboss.server.config.dir, pem=true)
+
+# create a truststore for the client called client.truststore and add it to the server config directory
+/subsystem=elytron/key-store=clientTS:add(path=client.truststore, relative-to=jboss.server.config.dir, credential-reference={clear-text=secret}, type=PKCS12)
```

Comment:
Since this is used on the client side, why is this CLI command needed?

Reply:
The user would need to take the server's certificate and configure their browser to trust it.

For the tests, the test's HTTP client would need to be configured to make use of the server's certificate.

Reply:
For manually testing the quickstart, did the user need to make use of the client's truststore previously? Or was it just the certificate that was needed?

Reply:
this is only used when running the tests. 
Like you mentioned when running the QS manually, we will be importing the certificate manually into the browser. But when running the test, we don't have a browser, so we need to make a client truststore and configure the client. 
This CLI script is preparing the client trustore for testing purposes only.

Reply:
In that case, I think you could programmatically create the truststore from the test itself (we have examples in the Elytron testsuite).

Reply:
I added code to do this inside the test. 
We should be able to remove the CLI script for client truststore and the corresponding restoring CLI script.

Reply:
Thanks, please go ahead and make the corresponding updates the script. Currently, the script also export's the server's cert to a file so we'll need to make sure we still do that so that someone trying it out from their browser can import that cert.

Reply:
Similar changes could then be made to helloworld-mutual-ssl.